scripts/completions/harnx.bash

```
@@ -73,6 +77,26 @@ _harnx() {
                     fi
                     return 0
                     ;;
+                --mcp-root)
+                    local oldifs
+                    if [[ -v IFS ]]; then
+                        oldifs="$IFS"
+                    fi
+                    IFS=$'\n'
+                    COMPREPLY=($(compgen -d "${cur}"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)